compiler-rt/lib/scudo/standalone/secondary.h

```diff
@@ -659,8 +721,18 @@ MapAllocator<Config>::tryAllocateFromCache(const Options &Options, uptr Size,
                                            FillContentsMode FillContents) {
   CachedBlock Entry;
   uptr EntryHeaderPos;
+  uptr MaxAllowedFragmentedPages;
+
+  if (LIKELY(!useMemoryTagging<Config>(Options))) {
```

Comment:
I realized you can simplify this even more:

  uptr MaxAllowedFragmentedPages = MaxUnreleasedCachePages;
  if (UNLIKELY(useMemoryTagging...)
     MaxAllowedFragmentedPages += CachedBlock::MaxReleasedCachePages;